apps/nft.blocksense.network/app/lib/client.ts

```diff
@@ -1,17 +1,9 @@
 import { createThirdwebClient } from 'thirdweb';
 
-const getEnvVar = (key: string): string => {
-  const value = process.env[key];
-  if (!value) {
-    throw new Error(`Missing environment variable: ${key}`);
-  }
-  return value;
-};
-
-const clientId = getEnvVar('NEXT_PUBLIC_CLIENT_ID');
-const smAddress = getEnvVar('NEXT_PUBLIC_NFT_SMART_CONTRACT_ADDRESS');
-const privateKey = getEnvVar('NEXT_PUBLIC_PRIVATE_KEY');
-const secretKey = getEnvVar('NEXT_THIRDWEB_SECRET_KEY');
+const clientId = process.env.NEXT_PUBLIC_CLIENT_ID!;
+const smAddress = process.env.NEXT_PUBLIC_NFT_SMART_CONTRACT_ADDRESS!;
+const privateKey = process.env.NEXT_PUBLIC_PRIVATE_KEY!;
+const secretKey = process.env.NEXT_THIRDWEB_SECRET_KEY!;
```

Comment:
It is best to read env vars with this function https://github.com/blocksense-network/blocksense/blob/main/libs/ts/base-utils/src/env/functions.ts#L10